jbpm-console-ng-rest/src/main/java/org/jbpm/console/ng/rest/ProcessImageResourceImpl.java

```diff
@@ -0,0 +1,183 @@
+package org.jbpm.console.ng.rest;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.Iterator;
+import java.util.List;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceUnit;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.io.IOUtils;
+import org.guvnor.common.services.project.model.GAV;
+import org.guvnor.m2repo.backend.server.GuvnorM2Repository;
+import org.guvnor.m2repo.service.M2RepoService;
+import org.jbpm.kie.services.impl.model.ProcessAssetDesc;
+import org.jbpm.process.audit.AuditLogService;
+import org.jbpm.process.audit.JPAAuditLogService;
+import org.jbpm.process.audit.NodeInstanceLog;
+import org.jbpm.process.svg.SVGImageProcessor;
+import org.jbpm.services.api.DeploymentService;
+import org.jbpm.services.api.ProcessService;
+import org.jbpm.services.api.RuntimeDataService;
+import org.jbpm.services.api.model.ProcessDefinition;
+
+@Path("/runtime/{deploymentId: [\\w\\.-]+(:[\\w\\.-]+){2,2}(:[\\w\\.-]*){0,2}}/process/{processDefId: [_a-zA-Z0-9-:\\.]+}")
+@ApplicationScoped
+public class ProcessImageResourceImpl {
+
+    /* REST information */
+
+    @Context
+    private HttpHeaders headers;
+
+    /* KIE information and processing */
+
+    @Inject
+    private GuvnorM2Repository repository;
+    
+    @Inject
+    private RuntimeDataService dataService;
+
+    /** AuditLogService **/
+    private static final String PERSISTENCE_UNIT_NAME = "org.jbpm.domain";
```

Comment:
Do we need to have this hard-coded in this class? If the code is in the jbpm-console-ng repository it should be consuming services from jbpm-kie-services instead of consuming directly the EMF, right?